services/app-api/types.ts

```diff
@@ -32,6 +43,22 @@ export interface Measure {
    * it is set on fetch, according to the metadata in measureList.ts.
    */
   autoCompleted?: boolean;
+  data?: {
+    DataSource?: DataSource[];
```

Comment:
This type is not fully correct; in addition to being an array of strings from the DataSource enum, DataSource can also be "CAHPS 5.1H". This will throw a bit of a wrench into the rest of our code, which so far doesn't account for that possibility.

```suggestion
    DataSource?: DataSource[] | "CAHPS 5.1H";
```